modules/apps/object/object-test/src/testFunctional/tests/dependencies/ListType_New_Picklist_44198_20230322182913922.json

```diff
@@ -5,13 +5,19 @@
 			"externalReferenceCode": "ac1421ef-f4ab-f990-e162-57f80aadbee4",
 			"key": "newitem",
 			"name": "newitem",
+			"nameMap": {
+				"en-US": "newitem"
+			},
 			"name_i18n": {
 				"en-US": "newitem"
 			},
```

Comment:
Double check the correct order here.

Reply:
To verify this order:
- I put nameMap after name_i18n
- Run sf locally
- SF changed the order to the current one (nameMap > name_i18n)